docs/sources/http_api/dashboard_versions.md

```diff
@@ -177,6 +179,118 @@ Status Codes:
 - **401** - Unauthorized
 - **404** - Dashboard version not found
 
+## Get dashboard version by dashboard UID
+
+`GET /api/dashboards/id/:dashboardId/versions/:uid`
+
+Get the dashboard version with the given version, for the dashboard with the given UID.
+
+**Example request for getting a dashboard version**:
+
+```http
+GET /api/dashboards/id/1/versions/1 HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+```
+
+**Example response**:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=UTF-8
+Content-Length: 1300
+
+{
+  "id": 1,
+  "dashboardId": 1,
```

Comment:
here the same as the comments on the other PR, if we request by UID, we should also have dashboarduid in the response as meta data

Reply:
I added UID! Also keeping dashboardId for now until we fully remove the old endpoint